account-management/src/test/java/com/aprilhorizon/accountmanagement/AccountManagementSystemTests.java

```diff
@@ -0,0 +1,53 @@
+package com.aprilhorizon.accountmanagement;
+
+
+import com.aprilhorizon.accountmanagement.models.AccountRequest;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.RestTemplate;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class AccountManagementSystemTests {
+
+    @Test
+    @Disabled
```

Comment:
There are [@Disabled annotations](https://junit.org/junit5/docs/5.0.0-M2/api/org/junit/jupiter/api/Disabled.html#:~:text=Annotation%20Type%20Disabled&text=%40Disabled%20is%20used%20to%20signal,are%20automatically%20disabled%20as%20well.) before each method in this class so none of the methods are executed during the test. Do we need this class? Maybe we can write all the test cases in `AccountManagementControllerTests` and remove `AccountManagementSystemTests`?